client/src/Header.js

```diff
@@ -60,7 +60,7 @@ const Header = () => {
 						</NavLink>
 						<NavLink
 							exact
-							to="/logout"
+							to="/"
```

Comment:
Did you test this? I think you might want to `preventDefault` so the user is not taken to `/` if there is an error logging out

Reply:
Yes. I have tested now.